packages/salesforcedx-vscode-core/package.json

```diff
@@ -734,6 +734,11 @@
           "type": "boolean",
           "default": false,
           "description": "%push_or_deploy_on_save_enabled_description%"
+        },
+        "salesforcedx-vscode-core.conflict-detection.enabled": {
```

Comment:
Micro-nit 😄, especially since other settings are doing the dash convention, but can we use camelCase for the property name? Capital letters (except the first letter) are interpreted by the settings UI as word breaks so "conflictDetection" will appear as "Conflict Detection: Enabled" in the UI instead of "conflict-detection: Enabled". Convention documented here: https://code.visualstudio.com/api/references/contribution-points#Configuration-schema

Reply:
I'll make the change. Seems like we can't change the others without effecting functionality?

Reply:
We should also remove the `enabled` word from it since we've gotten feedback from our users saying the way the settings UI displays the settings gives the impression the setting is already enabled when it isn't.
![Screen Shot 2020-02-04 at 1 47 01 PM](https://user-images.githubusercontent.com/1041842/73790250-059a1e00-4755-11ea-900f-be2f6a8f57d5.png)

Reply:
I changed it to just "conflictDetection" and the settings UI lost its mind. I'll need to work with this more. Will this setting change when the feature is GA?

Reply:
I'd probably only keep the word enabled if there are other sub-settings under the conflictDetection category. Changing the name of the setting I think would affect people who already have it set.

Reply:
This setting will probably be around for a while so we should try and pick a name we can stick to for the long run. @hbc-bindu can you help us choosing a name for this ?

Reply:
Some suggestions:
Manage Conflicts at Sync – most generic – detect, resolve, collect metrics and more
Resolve Conflicts at Sync – detect conflicts and allow resolution of the differences
Detect Conflicts at Sync - detect conflicts of the local/remote differences